README.md

```diff
@@ -195,6 +198,10 @@ You can use this view and the embedded _Export CSAF_ button to always quickly ex
 
 [(back to top)](#bsi-secvisogram-csaf-20-web-editor)
 
+### Templates
+
+The secvisogram provides some very basic templates. You can use your own templates by saving them as json files on your filesystem and load using the "Open" feature of the editor.
```

Comment:
Isn't it the 'Upload from filesystem' feature in  New -> 'Create new document' Dialog?

Reply:
Adapted :)